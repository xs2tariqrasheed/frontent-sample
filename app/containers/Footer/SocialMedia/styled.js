import styled from 'styled-components';

export const SocialLink = styled.a`
  font-family: Roboto;
  font-size: 30px;
  font-weight: normal;
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
  letter-spacing: -0.1px;
  text-align: right;
  margin-right: 20px;
  color: white;
  text-decoration: none;
`;
