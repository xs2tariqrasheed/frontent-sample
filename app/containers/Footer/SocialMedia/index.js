/*
 * Social Media
 *
 * Social media bar
 *
 */

import React from 'react';

import { SocialLink } from './styled';

export default () => (
  <div className="inline-block align-middle">
    <div className="md:text-right">
      <SocialLink href="https://twitter.com/BlockleteGames" target="_blank">
        <i className="fab fa-twitter" />
      </SocialLink>
      <SocialLink href="https://discord.gg/hy7cRwhCfW" target="_blank">
        <i className="fab fa-discord" />
      </SocialLink>
    </div>
  </div>
);
