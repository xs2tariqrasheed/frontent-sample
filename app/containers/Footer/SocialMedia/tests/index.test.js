import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';
import { Provider } from 'react-redux';
import { browserHistory, BrowserRouter } from 'react-router-dom';
import configureStore from '../../../../configureStore';
import SocialMedia from '../index';

describe('<SocialMedia />', () => {
  let store;

  beforeAll(() => {
    store = configureStore({}, browserHistory);
  });

  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <Provider store={store.store}>
        <BrowserRouter>
          <IntlProvider locale="en">
            <SocialMedia />
          </IntlProvider>
        </BrowserRouter>
      </Provider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
