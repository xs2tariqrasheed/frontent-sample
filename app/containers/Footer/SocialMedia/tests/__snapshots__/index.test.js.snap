// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SocialMedia /> should render and match the snapshot 1`] = `
<div
  class="col-span-12 md:col-start-11 md:col-span-2"
>
  <div
    class="md:text-right"
  >
    <a
      class="styled__SocialLink-sc-1984pln-0 jcYxES"
      href="https://twitter.com/BlockleteGames"
      target="_blank"
    >
      <i
        class="fab fa-twitter"
      />
    </a>
    <a
      class="styled__SocialLink-sc-1984pln-0 jcYxES"
      href="https://discord.gg/XDJAJdr"
    >
      <i
        class="fab fa-discord"
      />
    </a>
  </div>
</div>
`;
